go/plek.go

```diff
@@ -7,6 +7,8 @@ import (
 	"strings"
 )
 
+const DevDomain = "dev.gov.uk"
```

Comment:
As this is an exported constant, it needs a godoc comment, otherwise `golint` will complain.